src/Hastlayer/Hast.Catapult/CatapultLibrary.cs

```diff
@@ -551,4 +547,9 @@ private Task TesterWriteLineAsync(string format, params object[] objects)
         LogFunction?.Invoke((uint)level, text.ToString(CultureInfo.InvariantCulture) + Environment.NewLine);
 
     public override string ToString() => InstanceName;
+
+    private static object GetOrDefault(IDictionary<string, object> config, string key, object fallback = null) =>
+        config.TryGetValue(key, out var result)
+            ? result ?? fallback
+            : fallback;
```

Comment:
This looks like a good candidate for HL.

Reply:
On second thought 2 of the 3 usages is just `config.GetMaybe(key)` and the other is `config.GetMaybe(key) ?? fallback` so no need for a separate method.